.taskmaster/agent-terminals.py

```
@@ -0,0 +1,694 @@
+        pid_file = os.path.join(self.worktree_path, 'agent.pid')
+        running = False
+        pid = None
+        
+        if os.path.exists(pid_file):
+            try:
+                with open(pid_file, 'r') as f:
+                    pid = int(f.read().strip())
+                os.kill(pid, 0)  # Test if process exists
+                running = True
+                self.status = "running"
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)